lib/ansible/modules/cloud/amazon/s3_bucket.py

```diff
@@ -134,6 +134,12 @@
 except ImportError:
     HAS_BOTO = False
 
+try:
+    from functools import cmp_to_key
```

Comment:
Note that python-2.7 appears to have this function as well.  So we end up using the py3cmp function on python-2.7 as well.  Not a problem as the code should work there too, just something to be aware of.